From 91732e1fcb08814c495241a1f552d0d9080e26b6 Mon Sep 17 00:00:00 2001
From: ZhengShunQian <zhengsq@rock-chips.com>
Date: Tue, 22 Mar 2016 11:05:49 +0800
Subject: [PATCH] arm64: dts: rockchip: add compatible for rk3399evb board

Coreboot choose dtb by its compatible string.
Add "google,rk3399evb-rev*" accordingly.

Support more versions for rk3399evb in the future.
If we later find we need to introduce differences between versions,
it's easy to change things.

Change-Id: I049b4f113b1694577a1f0be68f6b635ae13653c0
Signed-off-by: ZhengShunQian <zhengsq@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-chrome.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts b/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
index 66d7636b4966..23a80bfa51b5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
@@ -45,7 +45,7 @@
 
 / {
 	model = "Rockchip Chrome";
-	compatible = "rockchip,chrome", "rockchip,rk3399";
+	compatible ="google,rk3399evb-rev2",  "google,rk3399evb-rev1", "google,rk3399evb-rev0", "rockchip,chrome", "rockchip,rk3399";
 
 	chosen {
 		bootargs = "rootwait cros_debug cros_secure ro root=/dev/mmcblk0p3 init=/sbin/init console=uart,mmio32,0xff1a0000 clk_ignore_unused";
-- 
2.35.3

